Update centralized-deployment-FAQ.md

msbemba opened the pull request:
Global admin is the only role that can deploy the add-in.. Tested with the roles that had this access earlier (Exchange admin and Office apps admin), But both don't have the integrated apps option anymore.

msbemba committed:
Update centralized-deployment-FAQ.md

Global admin is the only role that can deploy the add-in.. Tested with the roles that had this access earlier (Exchange admin and Office apps admin), But both don't have the integrated apps option anymore.

msbemba commented:
@JohanFreelancer9 Please copy review

msbemba commented:
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5243

JohanFreelancer9 commented on microsoft-365/admin/manage/centralized-deployment-FAQ.md:
```diff
@@ -92,8 +92,8 @@ We suggest reaching out to the ISV Developer for the paid add-in to request a ma
 ## Which admin role do I need to manage add-ins for my organization?  
 
-Global Admin is the recommended role with complete access to add-in management lifecycle. Other Admin roles have a limited access to add-in deployment lifecycle. If you're the person who purchased your Microsoft 365 for business subscription, you are the Global admin. 
+Global Admin is the recommended role with complete access to add-in management lifecycle. If you're the person who purchased your Microsoft 365 for business subscription, you are the Global admin. 
```
```suggestion
Global Admin is the recommended role with complete access to add-in management lifecycle. If you're the person who purchased your Microsoft 365 Business subscription, you are the Global admin. 
```

JohanFreelancer9 reviewed (changes requested):
Copy review - minor edit.

msbemba committed:
Update microsoft-365/admin/manage/centralized-deployment-FAQ.md

it should be microsoft 365 for business as that is the name of the subscription.

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@kwekuako Please merge this PR. Thanks.
